bedrock/stories/templates/stories/articles/joy-of-color.html

```diff
@@ -81,9 +81,9 @@
-    <p>The somber tones that plague digital design have a parallel to modern architecture: design in the service of efficiency, reduced to its most simple form. The modernist movement of the early 20th century turned sharply away from superfluous decoration, including maximalist “ornamentation” (think of turning away from centuries of maximalist decoration: chinoiserie, art nouveau, the arts & crafts movement, etc.). Lee referred to “Ornament and Crime,” a manifesto in which architect Adolph Loos decried the use of decoration for decoration’s sake, in favor of useful objects as influential in the shift toward  rectilinear buildings that were reduced to the most simplified form with no “unnecessary” features. The prevailing view was that only those who lacked restraint—children and people in non-Western societies—favored overtly decorative design fraught with unnecessary use of color (color used without purpose or intention).</p>
-    <p> “Holding back from ornamentation was a sign of spiritual strength, so if you paraded conspicuous decoration overwrought with an abundance of color, you were basically showcasing your own moral weakness,” said Lee.</p>
-    <p>The shift toward full blown modernism manifested the notion that “Modern” people stray away from conspicuous decoration and therefore overuse of color ,which became a trope in modern culture. Restraint use of color and minimalism became the status quo among those who considered themselves arbiters of taste. Architecture and interior design embraced a restrained use of color.</p>
+    <p>The somber tones that plague digital design have a parallel to modern architecture: design in the service of efficiency, reduced to its most simple form. The modernist movement of the early 20th century turned sharply away from superfluous decoration, including maximalist “ornamentation” (think of turning away from centuries of maximalist decoration: chinoiserie, art nouveau, the arts & crafts movement, etc.). Lee referred to “Ornament and Crime,” a manifesto in which architect Adolph Loos decried the use of decoration for decoration’s sake, in favor of useful objects as influential in the shift toward  rectilinear buildings that were reduced to the most simplified form with no “unnecessary” features. The theory was that only those who lacked restraint—children and people in non-Western societies—favored overtly decorative design fraught with unnecessary use of color (color used without purpose or intention).</p>
+    <p> “Holding back from ornamentation was a sign of spiritual strength, so if you paraded conspicuous decoration overwrought with an abundance of color, you were basically showcasing your own moral weakness,” said Lee referencing Loos’ essay.</p>
```

Comment:
Not requested, but my brain expected a comma

```
- said Lee referencing Loos’ essay.
+ said Lee, referencing Loos’ essay.
```

Reply:
There are some other copy edits we _could_ do but will Slack about them

Reply:
Adding comma, my brain was looking for the same pause